### PR TITLE
add unit test plot_heatmap #443

### PR DESCRIPTION
**Pull request by sonaalKant**
Issue #443

### CODE REVIEW

File: helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt [hunk 1]
```diff
@@ -1 +1 @@
-abc
+abc
```
Comment (DanilYachmenev) [inline helpers/test/outcomes/TestCheckDataFrame1.test_check_df_not_equal3/output/test_df.txt:1]: So these changes are actually happening due to the known issue

could you pls apply `amp/dev_scripts/clean_up_text_files.sh` on every file that has such 'changes'
they should revert to the previous stage and disappear from this PR

**Comment by sonaalKant** [comment]
@DanilYachmenev You can review now.

**Review by DanilYachmenev — COMMENTED** [review]
lgtm, fast and clear

**Review by gpsaggese — COMMENTED** [review]
Way to go

**Pull request merged**